infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntime.java

```diff
@@ -248,7 +250,8 @@ void createPods(List<Service> services, List<Route> routes) throws Infrastructur
                 container.getName(),
                 serverResolver.resolve(machineName),
                 project,
-                MachineStatus.STARTING);
+                MachineStatus.STARTING,
+                machineConfigs.get(machineName).getAttributes());
```

Comment:
This code must not throw npe, because all machines from  recipe are present in the internal environment after [this](https://github.com/eclipse/che/pull/8228/commits/2188781aa18fa77a02b0e7a05a041a5ff2cfc420) changes.